roles/splunk/tasks/remove_splunk.yml

```diff
@@ -11,4 +11,139 @@
     state: absent
   become: true
 
-# Todo: Remove user, group, THP/ulimit files, check for cron jobs/scripts and remove those if present
+- name: Block for non-root splunk user removal
+  block:
+    - name: Add nix splunk user
```

Comment:
Remove?

Reply:
Fixed, along with a few other names, in https://github.com/splunk/ansible-role-for-splunk/pull/161/commits/b70c6e5be6e4238a9968acf0d00512ef849ca656